README.md

```diff
@@ -1,2 +1,106 @@
 # in-toto
-in-toto is a series of scripts to protect the supply chain integrity.
+Protecting the software supply chain integrity
+
+in-toto guarantees that the end-user (or client) is able to verify that the entire development life cycle has been conducted as per the specified layout and that each of the functionaries (eg. developers) have performed the specified tasks and there haven't been any malicious changes in the files.
+
+in-toto requires a project layout that specifies the functionaries and the tasks they are supposed to perform.
+After each functionary performs his or her task a link metadata is generated.
+This metadata is used to verify the intermediate and final products with the project layout.
+
+
+## Getting Started
+
+### Install Dependencies
+ - [Python](www.python.org) in version 2.7 - crypto libraries require header files
+ - [OpenSSL](https://www.openssl.org/) - crypto libraries require header files
+ - [git](https://git-scm.com/) - version control system
+ - [pip](https://pip.pypa.io) - package installer tool
+ - [virtualenvs](http://docs.python-guide.org/en/latest/dev/virtualenvs/) - optional but strongly recommended
+
+### Installation
+```shell
+# Fetch in-toto sources
+git clone -b develop --recursive https://github.com/in-toto/in-toto.git
+
+# Change into project root directory
+cd in-toto
+
+# Install with pip in "develop mode"
+pip install -e .
```

Comment:
Technicallity, but I think it is called "editable" mode. Maybe we want to expand to "--editable" given that we use "--recursive" above.